File: server/Models/Ingredient.cs
namespace allSpiceCP8.Models;

public class Ingredient : RepoItem<int>
{
	// NOTE: commented out because RepoItem.cs replaces them for us since they're used on all the models
	// public int Id { get; set; }
	// public DateTime CreatedAt { get; set; }
	// public DateTime UpdatedAt { get; set; }
	public string Name { get; set; }
	public string Quantity { get; set; }
	public int RecipeId { get; set; }
}

File: client/src/models/Ingredient.js
import { DatabaseItem } from "./DatabaseItem.js";

export class Ingredient extends DatabaseItem {
  constructor(data) {
	super(data) //calls constructor on DataBaseItem

		// id, createdAt, updatedAt inherited from DatabaseItem

		this.name = data.name
		this.quantity = data.quantity
		this.recipeId = data.recipeId
  }
}
